docs/contributing/committing.rst

```diff
@@ -119,3 +124,11 @@ The changes are ready to be pushed to ``master`` now.
     $ # Push the commits!
     $ git push upstream master
     $ git branch -d pr/xxxx
+
+When you have made a mistake
+============================
+
+It's ok! Everyone makes mistakes. If you realise that recent merged changes
```

Comment:
I wasn't sure whether it was appropriate to create a separate section for that (and not give it an explicit name). I imagine this could grow in the future, with other troubleshooting/rollback guidelines.